### ISSUE TITLE
Problem installing on Windos

### ISSUE THREAD

**Issue by peterch405** [biolab/orange3-network#7]
The error message say I need to install MS C++ 10.0 which I already have installed on my system. If you try the installation through anaconda prompt with `pip install orange3-network` everything works fine except the add-ons isn't showing up in orange3.

```
You are using pip version 6.0.8, however version 8.0.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Collecting Orange3-Network
  Using cached Orange3-Network-1.1.2.tar.gz
    build_src
    building extension "orangecontrib.network._fr_layout" sources
    build_src: building npy-pkg config files
Requirement already satisfied (use --upgrade to upgrade): networkx>=1.10 in c:\python34\lib\site-packages (from Orange3-Network)
Requirement already satisfied (use --upgrade to upgrade): pyqtgraph>=0.9.10 in c:\python34\lib\site-packages (from Orange3-Network)
Requirement already satisfied (use --upgrade to upgrade): decorator>=3.4.0 in c:\python34\lib\site-packages (from networkx>=1.10->Orange3-Network)
Requirement already satisfied (use --upgrade to upgrade): numpy in c:\python34\lib\site-packages (from pyqtgraph>=0.9.10->Orange3-Network)
Installing collected packages: Orange3-Network
  Running setup.py install for Orange3-Network
    unifing config_cc, config, build_clib, build_ext, build commands --compiler options
    unifing config_fc, config, build_clib, build_ext, build commands --fcompiler options
    build_src
    building extension "orangecontrib.network._fr_layout" sources
    build_src: building npy-pkg config files
    No module named 'numpy.distutils.msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
    customize MSVCCompiler using build_ext
    No module named 'numpy.distutils.msvccompiler' in numpy.distutils; trying from distutils
    customize MSVCCompiler
    Missing compiler_cxx fix for MSVCCompiler
    customize MSVCCompiler using build_ext
    building 'orangecontrib.network._fr_layout' extension
    compiling C sources
    error: Microsoft Visual C++ 10.0 is required (Unable to find vcvarsall.bat).
    Complete output from command C:\Python34\python.exe -c "import setuptools, tokenize;__file__='C:\\Users\\chovanec\\AppData\\Local\\Temp\\pip-build-gqjgc8_8\\Orange3-Network\\setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record C:\Users\chovanec\AppData\Local\Temp\pip-39qph15q-record\install-record.txt --single-version-externally-managed --compile:
    running install

    running build

    running config_cc

    unifing config_cc, config, build_clib, build_ext, build commands --compiler options

    running config_fc

    unifing config_fc, config, build_clib, build_ext, build commands --fcompiler options

    running build_src

    build_src

    building extension "orangecontrib.network._fr_layout" sources

    build_src: building npy-pkg config files

    running build_py

    creating build

    creating build\lib.win32-3.4

    creating build\lib.win32-3.4\orangecontrib

    copying orangecontrib\__init__.py -> build\lib.win32-3.4\orangecontrib

    creating build\lib.win32-3.4\orangecontrib\network

    copying orangecontrib\network\community.py -> build\lib.win32-3.4\orangecontrib\network

    copying orangecontrib\network\network.py -> build\lib.win32-3.4\orangecontrib\network

    copying orangecontrib\network\readwrite.py -> build\lib.win32-3.4\orangecontrib\network

    copying orangecontrib\network\setup.py -> build\lib.win32-3.4\orangecontrib\network

    copying orangecontrib\network\snap.py -> build\lib.win32-3.4\orangecontrib\network

    copying orangecontrib\network\__init__.py -> build\lib.win32-3.4\orangecontrib\network

    creating build\lib.win32-3.4\orangecontrib\network\widgets

    copying orangecontrib\network\widgets\graphview.py -> build\lib.win32-3.4\orangecontrib\network\widgets

    copying orangecontrib\network\widgets\OWNxAnalysis.py -> build\lib.win32-3.4\orangecontrib\network\widgets

    copying orangecontrib\network\widgets\OWNxClustering.py -> build\lib.win32-3.4\orangecontrib\network\widgets

    copying orangecontrib\network\widgets\OWNxExplorer.py -> build\lib.win32-3.4\orangecontrib\network\widgets

    copying orangecontrib\network\widgets\OWNxFile.py -> build\lib.win32-3.4\orangecontrib\network\widgets

    copying orangecontrib\network\widgets\OWNxFromDistances.py -> build\lib.win32-3.4\orangecontrib\network\widgets

    copying orangecontrib\network\widgets\OWNxGenerator.py -> build\lib.win32-3.4\orangecontrib\network\widgets

    copying orangecontrib\network\widgets\OWNxInsideView.py -> build\lib.win32-3.4\orangecontrib\network\widgets

    copying orangecontrib\network\widgets\OWNxSNAP.py -> build\lib.win32-3.4\orangecontrib\network\widgets

    copying orangecontrib\network\widgets\__init__.py -> build\lib.win32-3.4\orangecontrib\network\widgets

    creating build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\airtraffic.net -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\airtraffic_items.tab -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\dicty_publication.net -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\dicty_publication_edges.tab -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\dicty_publication_items.tab -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\lastfm.net -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\lastfm.tab -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\lastfm_all.tab -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\lastfm_tags.tab -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\leu_by_genesets.net -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\leu_by_genesets_items.tab -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\leu_by_pmid.net -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\leu_by_pmid_items.tab -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\leu_huttenhower_a.net -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\leu_huttenhower_a_items.tab -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\makeNetworkList.py -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\mips_c2_cp_leu.net -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\mips_c2_cp_leu_items.tab -> build\lib.win32-3.4\orangecontrib\network\networks

    copying orangecontrib\network\networks\network_info.tab -> build\lib.win32-3.4\orangecontrib\network\networks

    creating build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\button_autoscale.png -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\button_rect.png -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\Category-Network.svg -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\Dlg_clear.png -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\Dlg_HideSelection.png -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\Dlg_Mark2Sel.png -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\Dlg_Sel2Mark.png -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\Dlg_SelisMark.png -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\Dlg_UnselectedNodes.png -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\NetworkAnalysis.svg -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\NetworkClustering.svg -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\NetworkExplorer.svg -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\NetworkFile.svg -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\NetworkFromDistances.svg -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\NetworkGenerator.svg -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\NetworkInsideView.svg -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\SNAP.svg -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    copying orangecontrib\network\widgets\icons\snap_logo.png -> build\lib.win32-3.4\orangecontrib\network\widgets\icons

    running build_ext

    No module named 'numpy.distutils.msvccompiler' in numpy.distutils; trying from distutils

    customize MSVCCompiler

    customize MSVCCompiler using build_ext

    No module named 'numpy.distutils.msvccompiler' in numpy.distutils; trying from distutils

    customize MSVCCompiler

    Missing compiler_cxx fix for MSVCCompiler

    customize MSVCCompiler using build_ext

    building 'orangecontrib.network._fr_layout' extension

    compiling C sources

    error: Microsoft Visual C++ 10.0 is required (Unable to find vcvarsall.bat).

    ----------------------------------------
    Command "C:\Python34\python.exe -c "import setuptools, tokenize;__file__='C:\\Users\\chovanec\\AppData\\Local\\Temp\\pip-build-gqjgc8_8\\Orange3-Network\\setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record C:\Users\chovanec\AppData\Local\Temp\pip-39qph15q-record\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in C:\Users\chovanec\AppData\Local\Temp\pip-build-gqjgc8_8\Orange3-Network

```

**Comment by kernc** [comment]
Please let us know if you figure it out! :pray:

**Comment by peterch405** [comment]
Seems to be a Microsoft vs version issue (can't get visual studio 2010 express anymore):
http://stackoverflow.com/questions/28251314/error-microsoft-visual-c-10-0-is-required-unable-to-find-vcvarsall-bat

Ended up just installing Orange3 through anaconda.

**Comment by kernc** [comment]
So did you have VS 2010 or did you have another version? Did you try any of the [other fixes](http://stackoverflow.com/questions/28251314/error-microsoft-visual-c-10-0-is-required-unable-to-find-vcvarsall-bat/32006750#32006750) (i.e. configuring the environment variables)? Does the Anaconda version install this add-on?

**Comment by peterch405** [comment]
I have Visual C++ 2010 version 10.0.40219 installed and I tried editing the environmental variable after I installed VS 2015 community edition, both of which didn't work. I attributed it to Orange3 getting installed with python 3.4 which is only supported with VS2010 whereas python 3.5 is reported to work with VS2015. I already had anaconda python 2.7 installed so I installed python 3.5 in a new environment (http://conda.pydata.org/docs/py2or3.html) and installed Orange3 from the anaconda pre-built packages (`conda install -c https://conda.anaconda.org/ales-erjavec orange3`). Then installing the addons with pip worked as I have reported before, but this time because it was installed in the same environment the addons showed up in orange3. The only downside is no desktop shortcut, but I can live with that.
